tools/Google.Cloud.Tools.Analyzers.Tests/DefaultLiteralTest.cs

```diff
@@ -78,6 +92,31 @@ public class A
         }
 
         [Fact]
+        public void DefaultNullableBoolLocal()
+        {
+            var test = @"
+public class A
+{
+    public void B()
+    {
+        bool? x = default;
+    }
+}";
+            var expected = CreateDiagnostic("bool?", 6, 19);
```

Comment:
I don't think the bug affects local variables - at least as far as I can see. It looks like it's only parameters.

Reply:
Yup, you're right. I just assumed it would. Fixed.